timeline-chart/src/time-graph-container.ts

```diff
@@ -45,14 +45,19 @@ export class TimeGraphContainer {
             backgroundColor: config.backgroundColor,
             transparent: config.transparent,
             antialias: true,
-            roundPixels: false,
             resolution: ratio,
-            autoResize: true
         });
         this.application.stage.height = config.height;
 
         this.stage = this.application.stage;
         this._canvas = this.application.view;
+        let app: PIXI.Application;
```

Comment:
If you don't plan to assign this variable again, const is preferred to let. Also, you can put the assignation on the same line and you may not need to  put the PIXI.Application (with proper eslint-typescript configuration, it will tell you if you do or don't).

Reply:
removed!